delegation-toolkit/tutorials/_category_.json

```diff
@@ -1,4 +0,0 @@
-{
-  "label": "Tutorials",
-  "position": 5
-}
```

Comment:
The link to global tutorial is missing like SDK. We can either push it in the PR, happy if we are fixing with a separate PR. Not a blocker. 
<img width="370" height="185" alt="Screenshot 2025-09-02 at 3 27 02 PM" src="https://github.com/user-attachments/assets/2f4b314e-5738-42db-848e-5edd0fae2995" />

Reply:
Dtk sidebar is autogenerated. You have manually make the sidebar in order to add links